src/syscon.rs

```diff
@@ -15,16 +15,7 @@ pub mod frg;
 #[cfg(feature = "845")]
 pub use self::frg::FRG;
 
-#[cfg(feature = "82x")]
-/// Clock configuration for peripherals
-pub mod clocksource_82x;
-#[cfg(feature = "82x")]
-pub use clocksource_82x as clocksource;
-#[cfg(feature = "845")]
-/// Clock configuration for peripherals
-pub mod clocksource_845;
-#[cfg(feature = "845")]
-pub use clocksource_845 as clocksource;
+pub mod clocksource;
```

Comment:
Nice!

Though I'm slightly confused that it works. I don't see any updates to the examples (and I assume this should break some of them). But CI is still green?

Reply:
In clocksource.rs I've added `pub use clocksource_845::*;`, which makes it mostly non-breaking

Reply:
I saw that, but I somehow had missed that these were already used `as clocksource` in `syscon` (yes, I see now that it's right here in the diff). Anyway, all makes sense now.